sharding/hashValidatorShuffler.go

```diff
@@ -657,6 +706,9 @@ func (rhs *randHashShuffler) UpdateShufflerConfig(epoch uint32) {
 		"epochEnable", rhs.activeNodesConfig.EpochEnable,
 		"maxNodesToShufflePerShard", rhs.activeNodesConfig.NodesToShufflePerShard,
 	)
+
+	rhs.flagBalanceWaitingLists.Toggle(epoch >= rhs.balanceWaitingListsEnableEpoch)
+	log.Trace("balanced waiting lists", "enabled", rhs.flagBalanceWaitingLists.IsSet())
```

Comment:
log.Debug as to align with the rest of the project.

Reply:
done